spec/outputs/dynatrace_spec.rb

```diff
@@ -232,18 +232,39 @@
     end
   end
 
-  context 'max_payload_size 2MB' do
-    let(:config) { { 'ingest_endpoint_url' => ingest_endpoint_url, 'api_key' => api_key, 'max_payload_size' => 2_000_000 } }
+  context 'max_payload_size 500' do
+    let(:config) { { 'ingest_endpoint_url' => ingest_endpoint_url, 'api_key' => api_key, 'max_payload_size' => 500 } }
     subject { LogStash::Outputs::Dynatrace.new(config) }
 
     before do
       allow(subject).to receive(:send_event) { |e, att| [:success, e, att] }
-      subject.multi_receive([1, 2].map { |n| LogStash::Event.new({ 'n' => n.to_s * 1_250_000 }) })
     end
 
-    it 'should split the chunk into multiple requests' do
+    it 'should send 2 400B messages in multiple requests' do
+      subject.multi_receive([1, 2].map { |n| LogStash::Event.new({ 'n' => n.to_s * 400 }) })
       expect(subject).to have_received(:send_event).exactly(2).times
     end
+
+    it 'should send 2 100B messages in a single request' do
+      subject.multi_receive([1, 2].map { |n| LogStash::Event.new({ 'n' => n.to_s * 100 }) })
+      expect(subject).to have_received(:send_event).exactly(1).time
+    end
+
+    it 'should drop messages larger than max_payload_size' do
+      subject.multi_receive([
+        LogStash::Event.new({ 'event' => '🤣' * 400 }),
+        LogStash::Event.new({ 'event' => 'n' * 600 }),
+        LogStash::Event.new({ 'event' => 'n' * 400 }),
+      ])
+      expect(subject).to have_received(:send_event).exactly(1).time
+    end
+
+    it 'should drop messages larger than max_payload_size' do
```

Comment:
```suggestion
    it 'should accept messages smaller than max_payload_size' do
```
I assume?

Reply:
Oh this is just a bad duplicate test. I'll remove it